README.md

```diff
@@ -102,14 +103,14 @@ typeof(v1[[1,2,3]]) <: Vector # Can't determine size from the type of [1,2,3]
 
 # Is (partially) hooked into BLAS, LAPACK, etc:
 rand(MMatrix{20,20}) * rand(MMatrix{20,20}) # large matrices can use BLAS
-eig(m3) # eig(), etc uses specialized algorithms up to 3×3, or else LAPACK
+eigvecs(m3) # eigvecs(), etc uses specialized algorithms up to 3×3, or else LAPACK
```

Comment:
FYI the equivalent thing is `eigen`. (It probably doesn't matter).

Reply:
Ah, I didn't find this, I only found `eigvecs` and `eigvals`. I will change it to `eigen`.